components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -390,7 +393,7 @@ export class ReportingComponent implements OnInit, OnDestroy {
   onFiltersClear(_event) {
     const queryParams = {...this.route.snapshot.queryParams};
 
-    const filteredParams = pickBy((_value, key) => {
+    const filteredParams = pickBy((_value, key: ReportingFilterTypes) => {
```

Comment:
I think I did this piece right, but i'm not positive.  Linter didn't like me passing a "string" as they type, and needed it to be specifically ReportingFilterTypes which is the enum where the Keys actually originate.